pages/profile/application/edit.tsx

```diff
@@ -0,0 +1,674 @@
+                    />
+                    {profile.resume && (
+                      <label className="cursor-pointer underline text-[#40B7BA]" htmlFor="resume">
+                        Replace resume
+                      </label>
+                    )}
+
+                    <br />
+                    {profile.resume && (
+                      <a
+                        href={profile.resume}
+                        target="_blank"
+                        className="text-[#40B7BA] underline cursor-pointer"
+                      >
+                        View current resume
+                      </a>
+                    )}
+                    <p className="poppins-regular text-xs text-[#40B7BA]">
+                      Accepted file types: .pdf, .doc, .docx, .png, .jpeg, .txt, .tex, .rtf
+                    </p>
+                  </div>
+                </section>
+              )}
+              {/* Teammate Questions */}
+              {registrationSection == 6 && (
+                <section className="bg-white lg:w-3/5 md:w-3/4 w-full min-h-[35rem] mx-auto rounded-2xl md:py-10 py-6 px-8 mb-8 text-[#4C4950]">
+                  <h2 className="sm:text-2xl text-xl font-semibold sm:mb-3 mb-1">
+                    Current Teammate
+                  </h2>
+                  {/* show teammate 1 if exists */}
+                  {profile.teammate1 && (
+                    <div className="flex flex-col poppins-regular md:px-4">
+                      <div className="flex items-center">
+                        <label className="font-semibold">Teammate 1:</label>
+                        <span className="ml-2">{profile.teammate1}</span>
+                      </div>
+                    </div>
+                  )}
+                  {/* show teammate 2 if exists */}
+                  {profile.teammate2 && (
+                    <div className="flex flex-col poppins-regular md:px-4">
+                      <div className="flex items-center">
+                        <label className="font-semibold">Teammate 2:</label>
+                        <span className="ml-2">{profile.teammate2}</span>
+                      </div>
+                    </div>
+                  )}
+                  {/* show teammate 3 if exists */}
+                  {profile.teammate3 && (
+                    <div className="flex flex-col poppins-regular md:px-4">
+                      <div className="flex items-center">
+                        <label className="font-semibold">Teammate 3:</label>
+                        <span className="ml-2">{profile.teammate3}</span>
+                      </div>
+                    </div>
+                  )}
+                  <p className="text-md my-6 font-bold">
+                    Emails of teammates should be the same as the email they registered with!
+                  </p>
```

Comment:
Maybe remove this text since we're not allowed user to change teammate email.

Reply:
we can still allow them to, but it's just that they have to email hello@hackutd.co and we change it manually on that admin side?